### PR TITLE
feat: #4759 venus-auth: add docker feature

### PR DESCRIPTION
**Pull request by LinZexiao**
## Related Issues
https://github.com/filecoin-project/venus/issues/4759

## Proposed Changes
add docker features
add github action for automaticly pushing to dockerhub

### CODE REVIEW

File: docker/script/compose.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+echo "regist admin"
+/app/venus-auth user add --name="admin"
+token=`/app/venus-auth token gen --perm admin admin`
+/app/venus-auth  user active admin
+
+echo "token:"
+echo ${token#*: }
+echo "${token#*: }" > /env/token
+
+wait
```
Comment (diwufeiwen) [inline docker/script/compose.sh:17]: 这类警告修一下

**Review by diwufeiwen — COMMENTED** [review]
build_upload.yml 中的push和pull_request的分支改一下就没问题了
 pull_request:
    branches:
      - main
  push:
    branches:
      - main
      - test*

File: docker/script/compose.sh (hunk 1 above)
Comment (LinZexiao) [reply docker/script/compose.sh]: 监听的主分支改成了 master，然后文末换行也ok了

**Review by diwufeiwen — APPROVED** [review]
LGTM

**Pull request merged**